pom.xml

```diff
@@ -29,6 +29,7 @@
         <dependency-check.version>8.4.0</dependency-check.version>
         <ossindex.version>3.2.0</ossindex.version>
         <jgit.version>6.8.0.202311291450-r</jgit.version>
+        <junit.version>5.9.2</junit.version>
```

Comment:
It turns out that the documentation recommends a different way of declaring the dependencies in Maven, so we may not need this property after all. Have a look at https://junit.org/junit5/docs/current/user-guide/#running-tests-build-maven

You'll see that you can use the `junit-bom` to dictate the version of all the JUnit dependencies. Bear in mind that you need to add that dependency inside our existing `dependencyManagement` section, not under the regular `dependencies` section. You still need to add the `junit-jupiter` and `junit-jupiter-params` dependencies to `dependencies`, like you're currently doing. Also notice that, according to the documentation, it looks like you need `junit-jupiter` rather than `junit-jupiter-engine`, apparently.